.github/comment.yml

```diff
@@ -122,8 +122,15 @@
     variables:
       openapiHub: https://aka.ms/openapihub
     keywords:
-      - "I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow."
-    onOpenedComments: "Hi, @${PRAuthor} Thanks for your PR. I am workflow bot for review process. Here are some small tips. </br> <li> Please ensure to do self-check against checklists in first PR comment.</li> <li> PR assignee is the person auto-assigned and responsible for your current PR reviewing and merging. </li>   <li> For specs comparison cross API versions,  Use [API Specs Comparison Report Generator](${openapiHub}/tools/diff?pr=${owner}/${repo}/${PRNumber})</li>  <li> If there is CI failure(s), to fix CI error(s) is mandatory for PR merging; or you need to provide justification in PR comment for explanation. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md)</li></br> Any feedback about review process or workflow bot, pls contact swagger and tools team. vscswagger@microsoft.com"
+      - "I have reviewed the general guidance on the spec PR review process: https://aka.ms/specprreview."
+    onOpenedComments: >- 
+      Hi, @${PRAuthor}! Thank you for your pull request. Here are some small tips to help you getting your PR merged.</br>
+      <li>Ensure you reviewed the checklists in the first PR comment.</li>
+      <li>PR assignee is the person auto-assigned and responsible for your current PR reviewing and merging.</li>
+      <li>For specs comparison across API versions, 
+      use [OpenAPIHub Specs Comparison Report Generator](${openapiHub}/tools/diff?pr=${owner}/${repo}/${PRNumber})</li>
+      <li>For convenient view of the API changes made by this PR, please refer to the URLs provided in the table in
```

Comment:
@maririos @praveenkuttappan I have few questions about this.

Let's look at this PR:

https://github.com/Azure/azure-rest-api-specs/pull/24101

This comment:
- https://github.com/Azure/azure-rest-api-specs/pull/24101#issuecomment-1558349231

points to:
[API Specs Comparison Report Generator](https://portal.azure-devex-tools.com/tools/diff?pr=Azure/azure-rest-api-specs/24101)

but it appears hard to use by me, because it has mandatory fields which I don't know how to fill out.

There is also the [Generated ApiView](https://github.com/Azure/azure-rest-api-specs/pull/24101#issuecomment-1558350428) but it doesn't have any diff in it, which appears to contradict what @praveenkuttappan mentioned on our Teams discussion.

It is possible to get the diff in APIView, as [seen here](https://github.com/Azure/azure-rest-api-specs/pull/24101#issuecomment-1558350428). But this info is nowhere to be found on the PR, @weshaggard linked me to it.

Hence this entire experience feel suboptimal to me.

How could we simplify all of it? Can we make the diff link in APIView immediately available? Do we still need the OpenAPI Hub API Specs Comparison Report Generator ?

Reply:
Diff is available in APIView. But I understand that a basic understanding of API View is required to view the diff. We can add a comment that points to APIView wiki. Here is the diff. https://apiview.dev/Assemblies/Review/f644c601fb9248388af7a0d9d1663983?diffRevisionId=ebe7178cc07f452da0b3daaca38a729f&diffOnly=False&revisionId=718c82f43dd143df85dc84c402bf8169&doc=False

Reply:
We haven't deprecated teh diff feature in open api hub so it may still be useful. But Idea is to utilize swagger diff in APIView more and more.

Reply:
@praveenkuttappan @maririos should we make this step one in deprecating this diff functionally and simply remove this comment completely (or at least the diff link), in favor of having folks using ApiView instead?

@konrad-jamrozik it might be worth talking to Mike about the usage of this in the triage meeting tomorrow.

Reply:
> @praveenkuttappan @maririos should we make this step one in deprecating this diff functionally and simply remove this comment completely (or at least the diff link), in favor of having folks using ApiView instead?

I wanted to remove it, but one of the differences is that APIView doesn't generate reviews for all existing versions of an API Spec, whicle the Specs Comparison Tool does. https://github.com/Azure/azure-sdk-tools/issues/6159
If this is no used or needed, then I am in favor of removing that comment and move people to use APIView

Reply:
I will discuss this with @mikekistler tomorrow at 10 AM PST. @praveenkuttappan is also working on some docs updates.

Reply:
Per our chat today, removed link to `API Specs Comparison Report Generator`. @praveenkuttappan are you planning to push some changes to this PR before we merge?

Reply:
@praveenkuttappan if you have pending changes, we can do it in a follow-up PR.